.github/workflows/devtools.yml

```diff
@@ -6,7 +6,7 @@ on:
   pull_request: # run on all PRs, not just PRs to a particular branch
 
 jobs:
-  integration:
+  build:
     runs-on: macos-latest # while macbots are much slower, linux reliably crashes running this
```

Comment:
is this still the case? there are a ton more Linux bots than there are Mac bots...in theory we'll be competing with other google or googlechrome projects for these

Reply:
It didn't work locally on my linux machine. I can poke at it more later.